### DISCUSSION TITLE
I have a problem to use my font in Windows 7.

### DISCUSSION THREAD

**Discussion by jinyoungyangTransmission** [cefsharp/CefSharp#4442] [General]
- **What version of the product are you using?**
- version 89 nuget.

- **What architecture x86 or x64?**
- x86
	
- **What version of .Net?**
- .Net framework 4.5.2
    
- **On what operating system?**
- Windows 7

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
- Winform
    
- **What steps will reproduce the problem?**
- Use CefSharp in Windows 7 and put some web font to show. The browser couldnt download the font so it just display default font, which is Malgun Gothic for me.

- **What is the expected output? What do you see instead?**
- I want to see the html text with my fonts, but it just show on Malgun Gothic font

- **Does this problem also occur in the `CEF` Sample Application**
- because I use Windows 7, it is not possible to use 110 version.

**Comment by amaitland** [comment]
This is likely issue https://github.com/chromiumembedded/cef/issues/2858

> * because I use Windows 7

`Windows 7` is no longer supported.
